src/ServiceControl.Audit.Persistence.Tests.RavenDb5/PersistenceTestsConfiguration.cs

```diff
@@ -36,7 +36,7 @@ public Task Configure(Action<PersistenceSettings> setSettings)
                 IsSetup = true
             };
 
-            settings.PersisterSpecificSettings["ServiceControl/Audit/RavenDb5/UseEmbeddedInstance"] = UseEmbeddedInstance.ToString();
+            settings.PersisterSpecificSettings["ServiceControl/Audit/RavenDb5/useembeddedinstance"] = UseEmbeddedInstance.ToString();
```

Comment:
was this meant to test the change below (`new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase)`)? If that's the case, shall we revert this?